src/components/Candidate.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from 'styled-components';
+import { url } from './url';
+import axios from 'axios';
+
+const Candidate = ({ candidateName, voteCount, id, getCandidates }) => {
+  const getVote = async (id) => {
```

Comment:
`updateVoteCount`라는 네이밍이 더 좋을 것 같아요~

```suggestion
  const updateVoteCount= async (id) => {
```